sql/delete_all_data.sql

```diff
@@ -58,6 +58,8 @@ TRUNCATE TABLE OH_SUPPLIER;
 TRUNCATE TABLE OH_THERAPIES;
 TRUNCATE TABLE OH_VERSION;
 TRUNCATE TABLE OH_VISITS;
+TRUNCATE TABLE OH_TELEMETRY;
+TRUNCATE TABLE OH_USER_SETTINGS;
```

Comment:
aren't there 2 tables missing:   OH_DICOM_DATA and OH_HELP?

Reply:
OH_DICOM_DATA yes.
OH_HELP has never been used so far, but we can add.
Thanks!